rubygem/lib/zeus.rb

```diff
@@ -1,6 +1,11 @@
 # encoding: utf-8
 require 'socket'
 
+# load exact json version from Gemfile.lock to avoid conflicts
+gemfile = "#{ENV["BUNDLE_GEMFILE"] || "Gemfile"}.lock"
+if File.exist?(gemfile) && version = File.read(gemfile)[/^    json \((.*)\)/, 1]
```

Comment:
In this case I think we do actually want to match a literal space whereas `[[:space:]]` also matches whitespace like newlines.